grant/projects/subview.md

```diff
@@ -0,0 +1,99 @@
+# subview.xyz
+
+## **Project Description**
+
+Subview is blockchain explorer for inspecting and analyzing EVM Chains. Subview provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine).
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Crab DVM currently does not have a specific DVM block explorer. For developers, will encounter obstacles when using the Crab network, especially when developers develop a Dapp. Subview can provide detailed block information and transaction information，developers can read and verify Solidity smart contracts and access pre-existing contracts to fast-track development.
+
+2. For users, Subview support for ERC20 and ERC721 tokens, you can easily view your asset statistics, transaction history, and gas fee usage, contract method calls and logs are decoded to provide additional information directly from the UI.
+
+### **How you will integrate this project into Crab:**
+
+Subview will deploy a Crab Network based explorer instance, which will scan past blocks and complete all block statistics. Once the scan is complete, crab users visit the subview to query the blockchain data they need.
+
+### **Team**
+
+- Shell ( Front-end Engineer )
+- Eason ( DB Engineer )
+
+### **Legal Structure**
+
+Currently Subview has no defined legal structure other then a Dapp created by Subview team.
+
+## **Team Code Repos**
+
+The version is not open source.
```

Comment:
Our grant program has open-source requirement for what you apply for the grant

Reply:
ok, We are preparing, and will submit the relevant grant code next

Reply:
https://github.com/SubviewOrg/subview-crab-chain

Reply:
Thanks. It seems lack of open source license file in the repo?

Reply:
https://github.com/SubviewOrg/subview-crab-chain/blob/master/explorer/LICENSE

updated

Reply:
okay